_posts/2022-03-15-kubeflow-1.5-release.md

```diff
@@ -0,0 +1,50 @@
+
+## Installation, tutorials and documentation
+
+The Kubeflow v1.5 release process includes improvements to Kubeflow’s installation, tutorial and documentation.  For installation, the Manifest Working Group’s [documentation] (https://github.com/kubeflow/manifests) provided the starting place for the Kubeflow [Distributions](https://www.kubeflow.org/docs/started/installing-kubeflow/), and this collaboration has increased the maturity of the code and documentation for Kubeflow’s GitOps installation pattern.  The v1.5 release process included a tracking [issue]  (https://github.com/kubeflow/website/issues/3130) for the critical web pages that are updated in each release.  Additionally, there are many Working Group documentation updates and we invite you to open issues when you find documentation that needs improvement.  Kubeflow v1.5 also includes two tutorials, which are described below:
+
+**Tutorial 1** - [E2E MNIST with Kubeflow Tutorial](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb),which provides an end-to-end test sequence i.e. start a notebook, run a pipeline, execute training, hyperparameter tuning and model serving. 
+
+**Tutorial 2** - [Deploy and run your first interfere service with KServe Tutorial](https://github.com/js-ts/website-1/blob/master/content/en/docs/external-add-ons/kserve/first_isvc_kserve.ipynb) Run your first interface service on Kubeflow 1.5 i.e. define, create and check an InferenceService, post an interfere request and receive response.  Optionally run performance tests.
+ 
+## Join the community
+
+The v1.5 Release Team would like to thank everyone for their efforts on Kubeflow v1.5, especially the users, code contributors, working group leads, and distribution team leads. As you can see from the extensive contributions to Kubeflow 1.5, the Kubeflow Community is vibrant and diverse, and solving real world problems for organizations around the globe.
+
+Excited to contribute your great ideas? The Kubeflow Community [Working Groups](https://github.com/kubeflow/community/blob/master/wg-list.md) hold open meetings, public lists, and are always looking for more volunteers and users to unlock the potential of machine learning. If you’re interested in becoming a Kubeflow contributor, please feel free to check out the resources below. We look forward to working with you!
+
+- Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow Github page](https://github.com/kubeflow)
+- Join the [Kubeflow Slack Channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+- Join the [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) mailing list
+- Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/) 
```

Comment:
link  to youtube video of v1.5 presentation

Reply:
Also if possible adding links to kubeflow social media might be helpful

Reply:
I thought this request might fit on line 42 as proposed above